File: Java/Mahasiswa.java
public class Mahasiswa{
	//atribut private
	private String nama;
	private String nim;
	private String prodi;
	private String fakultas;

	//constructor
    public Mahasiswa(){
        this.nama = "";
        this.nim = "";
        this.prodi = "";
        this.fakultas = "";
	}

	//getter dan setter
    public void setNama(String nama){
		this.nama = nama;
	}

	public String getNama(){
		return this.nama;
	}

	public void setNim(String nim){
		this.nim = nim;
	}

	public String getNim(){
		return this.nim;
	}

	public void setProdi(String prodi){
		this.prodi = prodi;
	}

	public String getProdi(){
		return this.prodi;
	}

	public void setFakultas(String fakultas){
		this.fakultas = fakultas;
	}

	public String getFakultas(){
		return this.fakultas;
	}
}

File: C++/Mahasiswa.cpp
#include <string>
#include <iostream>

using namespace std;

class Mahasiswa{
	
	private:
		string nama;
		string nim;
		string prodi;
		string fakultas;

	public:
		Mahasiswa(){
            this->nama = "";
            this->nim = "";
            this->prodi = "";
            this->fakultas = "";
		}

		//getter dan setter
        void setNama(string nama){
			this->nama = nama;
		}

		string getNama(){
			return this->nama;
		}

		void setNim(string nim){
			this->nim = nim;
		}

		string getNim(){
			return this->nim;
		}

		void setProdi(string prodi){
			this->prodi = prodi;
		}

		string getProdi(){
			return this->prodi;
		}

		void setFakultas(string fakultas){
			this->fakultas = fakultas;
		}

		string getFakultas(){
			return this->fakultas;
		}

		~Mahasiswa(){
		}
};